I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scene1.rpy", line 4, in script
    yum "Ughhh, why did they schedule art history for the early hours of the goddamn morning..."
IOError: Couldn't find file 'c-1-1.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scene1.rpy", line 4, in script
    yum "Ughhh, why did they schedule art history for the early hours of the goddamn morning..."
  File "E:\renpy-6.99.12.4-sdk\renpy\ast.py", line 652, in execute
    renpy.exports.say(who, what, interact=self.interact, *args, **kwargs)
  File "E:\renpy-6.99.12.4-sdk\renpy\exports.py", line 1199, in say
    who(what, *args, **kwargs)
  File "E:\renpy-6.99.12.4-sdk\renpy\character.py", line 1016, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "E:\renpy-6.99.12.4-sdk\renpy\character.py", line 817, in do_display
    **display_args)
  File "E:\renpy-6.99.12.4-sdk\renpy\character.py", line 566, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "E:\renpy-6.99.12.4-sdk\renpy\ui.py", line 287, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "E:\renpy-6.99.12.4-sdk\renpy\display\core.py", line 2649, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "E:\renpy-6.99.12.4-sdk\renpy\display\core.py", line 3318, in interact_core
    renpy.audio.audio.periodic()
  File "E:\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 988, in periodic_thread_main
    periodic_pass()
  File "E:\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 932, in periodic_pass
    c.periodic()
  File "E:\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 416, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "E:\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "E:\renpy-6.99.12.4-sdk\renpy\loader.py", line 555, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'c-1-1.mp3'.

Windows-8-6.2.9200
Ren'Py 7.0.0.196
Ah!! My Roommate is a Succubus Hellbent on World Conquest!! 1.0
Tue Oct 23 23:57:47 2018
